src/components/Home/styles.module.scss

```diff
@@ -3,54 +3,83 @@
 .container {
   display: flex;
   flex-direction: column;
-  width: 100%;
   height: 100%;
+  max-width: 1480px; 
+  margin: 0 auto; 
 }
 
 .contentContainer {
   width: 100%;
-  height: 1000px;
+  height: 1500px;
```

Comment:
Same question about a discrete height

Reply:
Could you abstract this into a shared variable and add a comment of why you're choosing 1500px?